packages/esm-patient-list-app/src/index.ts

```diff
@@ -25,6 +26,17 @@ function setupOpenMRS() {
   const route = `patient-list`;
   const spaBasePath = `${window.spaBase}/${route}`;
 
+  async function getName(x: string) {
+    const cachedName = localStorage.getItem(x);
```

Comment:
Hi @Jexsie , let's not use the local storage for caching the name. The name stored in the local storage can become stale if any change happens in the server.